### PR TITLE
Update database_maintenance_tools.md

### PR DESCRIPTION
**Pull request by jahway603**
Included a blog post by Jackson Chen, which DID work when I followed it to perform Matrix Synapse Maintenance, versus the 2020 blog post by Victor Berger, which DID NOT work when performining maintenance.

See https://pr15083--synapse-docs-previews.netlify.app/usage/administration/database_maintenance_tools.html .

### Pull Request Checklist

* [X] Pull request is based on the develop branch
* [X] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [X] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
* [X] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

File: docs/usage/administration/database_maintenance_tools.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-This blog post by Victor Berger explains how to use many of the tools listed on this page: https://levans.fr/shrink-synapse-database.html
+This blog post by Jackson Chen explains how to use many of the tools listed on this page: https://jacksonchen666.com/posts/2022-12-03/14-33-00/
+This blog post by Victor Berger is from 2020 & outdated, so please only use as light reference https://levans.fr/shrink-synapse-database.html
```
Comment (DMRobertson) [inline docs/usage/administration/database_maintenance_tools.md:2]:
```suggestion
This blog post by Victor Berger is from 2020 & outdated, so please only use as light reference: https://levans.fr/shrink-synapse-database.html```

Can you describe how Berger's post did not work for you?

**Review by DMRobertson — DISMISSED** [review]
As well as the below, can you add a newsfile please?

File: docs/usage/administration/database_maintenance_tools.md (hunk 1 above)
Comment (jahway603) [reply docs/usage/administration/database_maintenance_tools.md:2]: I don't know what a newsfile is, but can add one when I understand what it is.

Comment (DMRobertson) [reply docs/usage/administration/database_maintenance_tools.md:2]: A newsfile is the same thing as a "changelog file". See the checklist in the PR description for more information.

Comment (jahway603) [reply docs/usage/administration/database_maintenance_tools.md]: This PR contains a newsfile which also explains why this PR is being created in the first place.

Comment (jahway603) [reply docs/usage/administration/database_maintenance_tools.md]: @DMRobertson do I need to add any additional files to this PR?

**Comment by jahway603** [comment]
@DMRobertson what is the next step? is there anything else I need to submit with this PR?

**Comment by DMRobertson** [comment]
> @DMRobertson what is the next step? is there anything else I need to submit with this PR?

No, thanks.  But:

> after I had experienced tons of frustration not getting anything to work in the ancient blog post from 2020.

it would be useful if you could describe _what_ your frustrations were with the old blog post, e.g. what commands failed and what error messages you saw; what .

(Additionally: there's no need to ping `@DMRobertson` twice; the Synapse team will see updates to this issue---though you're extremely unlikely to see a response over a weekend.)

File: changelog.d/15083.doc [hunk 2]
```diff
@@ -0,0 +1 @@
+Update database maintenance tools documentation to include a recent blog post (originally written on Dec 3, 2022 and continues to receive updates, last update on Feb 7, 2023) to replace the commonly suggested blog post from 2020 which commands do not work. This new recent blog post was shared by its author in the Matrix Synapse Admin room after I had experienced tons of frustration not getting anything to work in the ancient blog post from 2020, so I'm updating this documentation to help others not get completely frustrated when maintaining their homeservers. Contributed by @jahway603.
```
Comment (DMRobertson) [inline changelog.d/15083.doc:1]:
```suggestion
Refer to a more recent blog post on the [Database Maintenance Tools](https://matrix-org.github.io/synapse/latest/usage/administration/database_maintenance_tools.html) page. Contributed by @jahway603.
```

**Pull request merged**